File: structy/python/maxValue.py
def max_value(nums):
    max = float('-inf')
    for i in range(len(nums)):
        if nums[i] > max:
            max = nums[i]
    return max

# print(max_value([1,3,34, 100, 41, 29, 11, 24, 53, 22, 12]))

File: structy/javascript/maxValue.js
const maxValue = (nums) => {
    // todo
    let max = -Infinity
    for (let i = 0; i < nums.length; i++) {
      if (nums[i] > max) max = nums[i]
    }
    return max
  };

console.log(maxValue([1,2,3]))
